internal/controller/controller.go

```diff
@@ -107,6 +108,10 @@ func (c *Controller) Handle(n *node) error {
 		prefix := prefixParts[0]
 		provider, err := c.Providers.Get(prefix)
 
+		az := n.Labels["failure-domain.beta.kubernetes.io/zone"]
```

Comment:
The Kubernetes documentation suggests these labels have been deprecated in favour of those prefixed with `topology.kubernetes.io/`: https://kubernetes.io/docs/reference/labels-annotations-taints/

Reply:
resolved in 3e3165c